src/main/java/io/github/cottonmc/energy/api/ElectricalEnergyType.java

```diff
@@ -1,20 +1,35 @@
 package io.github.cottonmc.energy.api;
 
+import net.minecraft.text.TextComponent;
+import net.minecraft.text.TranslatableTextComponent;
+
 public class ElectricalEnergyType implements EnergyType {
 
-    private final int size;
+    private final int lvPerPacket;
 
-    public ElectricalEnergyType(int maxSize) {
-        this.size = maxSize;
+    public ElectricalEnergyType(int lvPerPacket) {
+        this.lvPerPacket = lvPerPacket;
     }
 
     public int getPacketSize() {
```

Comment:
This isn't an override; since you changed the packet sizing for electrical energy you can just get rid of this method.

Reply:
This constructor isn't for packet sizes. It's to determine the transformer conversion ratio between packets of different voltages. 1 MV packet turns into potentially many LV packets.

Reply:
I think the packet's value should still be carried as an amount of WU, not as an amount of LV packets.

Reply:
done.